glider_dac/erddap/templates/private/dataset.deployment.xml

```diff
@@ -37,6 +37,7 @@
         <att name="title">{{dataset_id}}</att>
         <att name="ioos_dac_checksum">{{checksum}}</att>
         <att name="ioos_dac_completed">{{completed}}</att>
+        <att name="gts_ingest">{{gts_ingest}}</att>
```

Comment:
Not something we're going to have to change right away, but rather just a note.  Prod/dev Glider DAC use templates that are not tracked by Git, or that is to say, they reside outside the project directory.  We may want to change this over to use these ones.